database/src/main/scala/cromwell/database/slick/tables/JobStoreComponent.scala

```diff
@@ -32,7 +30,7 @@ trait JobStoreComponent {
 
   protected val jobStore = TableQuery[JobStoreEntries]
 
-  val jobStoreAutoInc = jobStore returning jobStore.map(_.jobStoreTableId)
+  val jobStoreAutoInc = jobStore returning jobStore.map(_.jobStoreId) into ((a, id) => a.copy(jobStoreId = Option(id)))
```

Comment:
Get rid of the `into`, and just have `val jobStoreAutoInc = jobStore returning jobStore.map(_.jobStoreId)`.

In `addJobStoreEntry()`, you're only using the `JobStoreEntry.jobStoreId`, not the whole `JobStoreEntry`.